src/Epochs.jl

```diff
@@ -445,12 +453,20 @@ end
 
 Epoch{S, T}(ep::Epoch{S, T}) where {S, T} = ep
 
-function isapprox(a::Epoch, b::Epoch)
-    a.epoch == b.epoch && a.offset ≈ b.offset
+function isapprox(a::Epoch, b::Epoch; atol::Real=0, rtol::Real=atol>0 ? 0 : √eps())
```

Comment:
@bgodard I have implemented a more precise method of comparison (see below) which should deal with the first problem you mentioned.